src/styles.scss

```diff
@@ -1 +1,4 @@
 /* You can add global styles to this file, and also import other style files */
+
+html, body { height: 100%; }
```

Comment:
We can add global styling variables for colors and such. That way they can be imported and used throughout our app